src/documentation/netex_idfm_to_ntfs.md

```diff
@@ -195,81 +195,86 @@ comment_id | Line/noticeAssignments/NoticeAssignment/NoticeRef | This field is p
+
+All ServiceJourneyPattern of a `route` are stored as complementary `object_codes`. ServiceJourneyPattern nodes are listed in the nodes `CompositeFrame/frames/GeneralFrame[]/members/ServiceJourneyPattern[]`. The `GeneralFrame` to be used is the one with the `@ref` containing `NETEX_STRUCTURE`.
 
 NTFS field | Netex-IDFM element | Mapping rule/Comment
 --- | --- | ---
 object_type |  | fixed value `route` 
-object_id | *ServiceJourneyPattern/RouteRef/@ref* | This field is prefixed. 
+object_id | ServiceJourneyPattern/RouteRef/@ref | This field is prefixed. 
 object_system |  | fixed value `Netex_ServiceJourneyPattern`
-object_code | *ServiceJourneyPattern/@id* | The value of this field is used without transformation.
+object_code | ServiceJourneyPattern/@id | The value of this field is used without transformation.
 
 
 ### trips.txt
-`trips` are described in each **offre_** file in the nodes **ServiceJourney**. 
+`trips` are described in each **offre_*.xml** file in the nodes **CompositeFrame/frames/GeneralFrame[]/members/ServiceJourney[]**. 
+The `GeneralFrame` to be used is the one with the `@ref` containing `NETEX_STRUCTURE`.
 
 NTFS field | Netex-IDFM element | Mapping rule/Comment
 --- | --- | ---
-route_id | *ServiceJourney/JourneyPatternRef/@ref* | route_id of the Route containing the JourneyPatternRef as an object_code.
+route_id | ServiceJourney/JourneyPatternRef/@ref | route_id of the Route containing the `JourneyPatternRef` as an object_code.
 service_id |  | Defined using `ServiceJourney/DayTypeRef`, see (calendar.txt and calendar_dates.txt)[]
-trip_id | *ServiceJourney/@id* | This field is prefixed. 
-trip_headsign | *ServiceJourney/DestinationDisplayRef* | Content of the `DestinationDisplay/FrontText` node. If not available, the name of the `stop_point` of the last `stop_time` is used.
-trip_short_name | *ServiceJourney/DestinationDisplayRef* | Content of the `DestinationDisplay/PublicCode` node. If not available, this field is empty.
-company_id | *ServiceJourney/OperatorRef* | if *ServiceJourney/OperatorRef* is not defined, use *Line/OperatorRef* in [*lines.xml*](#linestxt) file. This field is prefixed.
-physical_mode_id | *Line/TransportMode* | see physical_modes definition
+trip_id | ServiceJourney/@id | This field is prefixed. 
+trip_headsign | ServiceJourney/DestinationDisplayRef | Content of the `DestinationDisplay/FrontText` node. If not available, the name of the `stop_point` of the last `stop_time` is used.
+trip_short_name | ServiceJourney/DestinationDisplayRef | Content of the `DestinationDisplay/PublicCode` node. If not available, this field is empty.
+company_id | ServiceJourney/OperatorRef | if `ServiceJourney/OperatorRef` is not defined, use `Line/OperatorRef` in [*lines.xml*](#linestxt) file. This field is prefixed.
+physical_mode_id | Line/TransportMode | see physical_modes definition
```

Comment:
Do we want to mention this field is **not** prefixed?

Reply:
It's stated in the corresponding chapter, but I've added it here and at the line/commercial_mode_id level too as a reminder ;)

Reply:
I wonder if these details should actually be left only for the file `common.md` (see #438).

Reply:
the definition of the mapping between netex value and the physical_mode values should be in this file, because each format is different. Maybe I'm missing something ?

Reply:
I was talking only about the prefix part. Obviously this section has to be there. But in order to know if you need (or not), to prefix something, this is actually described in #438 so I was thinking, maybe we can left out of this specification all details related to prefixes, except for a link to the `common.md`?

Reply:
Ok, I think I understand what you mean. There is indeed a few duplicated information,  but in this case (prefixes), it's very important and having it at both locations seems relevant IMHO. What do you think ?

Reply:
I'm fine with it if you think it's best to emphasize it.